alpine/APKBUILD

```diff
@@ -0,0 +1,67 @@
+# this is a slimmed-down version of https://git.alpinelinux.org/aports/tree/main/varnish/APKBUILD
+# Contributor: Pedro Filipe <xpecex@outlook.com>
+# Contributor: Leonardo Arena <rnalrd@alpinelinux.org>
+# Contributor: V.Krishn <vkrishn4@gmail.com>
+# Contributor: Natanael Copa <ncopa@alpinelinux.org>
+pkgname=varnish
+pkgver=@VERSION@
+pkgrel=1
+pkgdesc="High-performance HTTP accelerator"
+url="http://www.varnish-cache.org/"
+arch="all"
+license="BSD"
+options="!check" # tests pass, but ran before packaging
+depends="gcc libc-dev libgcc"
+depends_dev=
+makedepends="$depends_dev pcre-dev ncurses-dev libedit-dev py-docutils
+	linux-headers libunwind-dev python py-sphinx"
+install="$pkgname.pre-install"
+subpackages="$pkgname-dbg $pkgname-dev $pkgname-doc $pkgname-libs"
+pkgusers="varnish"
+pkggroups="varnish"
+source="$pkgname-$pkgver.tar.gz"
+
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	./configure \
+		--build="$CBUILD" \
+		--host="$CHOST" \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--infodir=/usr/share/info \
+		--localstatedir=/var/lib \
+		--without-jemalloc \
+		--with-unwind
+	make
+}
+
+check() {
+	cd "$builddir"
+
+	make -j12 check VERBOSE=1
+}
+
+package() {
+	cd "$builddir"
+
+	make DESTDIR="$pkgdir" install
+
+	local i; for i in $source; do
```

Comment:
Did I miss something or `$source` only contains the release archive?

Reply:
right, the orginal (https://git.alpinelinux.org/aports/tree/main/varnish/APKBUILD) has configuration files too, but we don't here (should we?) so that can go

Reply:
done